cmake/toolchain_mcu_artik053.cmake

```diff
@@ -16,7 +16,7 @@ set(CMAKE_SYSTEM_NAME TizenRT)
 set(CMAKE_SYSTEM_PROCESSOR armv7l)
 set(CMAKE_SYSTEM_VERSION ARTIK053)
 
-set(FLAGS_COMMON_ARCH -mcpu=cortex-r4 -mfpu=vfpv3 -fno-builtin -fno-strict-aliasing -fomit-frame-pointer -fno-strength-reduce -Wall -Werror -Wshadow -Wno-error=conversion)
+set(FLAGS_COMMON_ARCH -mcpu=cortex-r4 -mthumb -mfpu=vfpv3 -fno-builtin -fno-strict-aliasing -fomit-frame-pointer -fno-strength-reduce -Wall -Werror -Wshadow -Wno-error=conversion)
```

Comment:
This line is way too long. It would be better to make it shorter, e.g:
```
set(FLAGS_COMMON_ARCH -mcpu=cortex-r4 -mthumb -mfpu=vfpv3
                      -fno-builtin -fno-strict-aliasing -fomit-frame-pointer -fno-strength-reduce
                      -Wall -Werror -Wshadow -Wno-error=conversion)
```